Integration/HousekeepingLeuchtfeuerIntegration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\LeuchtfeuerHousekeepingBundle\Integration;
+
+use Mautic\IntegrationsBundle\Integration\BasicIntegration;
+use Mautic\IntegrationsBundle\Integration\ConfigurationTrait;
+use Mautic\IntegrationsBundle\Integration\Interfaces\BasicInterface;
+
+class HousekeepingLeuchtfeuerIntegration extends BasicIntegration implements BasicInterface
+{
+    use ConfigurationTrait;
+
+    public const INTEGRATION_NAME = 'housekeepingleuchtfeuer';
```

Comment:
See above (or is the namespace wrong? I'd expect `leuchtfeuerhousekeeping` here as name)

Reply:
See my comment before.